### PR TITLE
Don't truncate URLs in formatted output

### PR DESCRIPTION
**Pull request by jeremy**
## Summary

- `formatCell` truncates any string over 40 runes, which destroys URL fields like `todolists_url` and `app_todolists_url` — a truncated URL can't be clicked, copied, or navigated
- Add `isURL()` helper to skip truncation for `http://` and `https://` prefixes
- Affects all four render paths (styled/markdown × object/table) since they all funnel through `formatCell`

## Test plan

- [x] Unit tests: long URL preserved, non-URL still truncated, short URL passthrough, http scheme
- [x] Regression tests: styled and markdown renderers preserve full URLs in both object and table views
- [x] `bin/ci` green

---
## Summary by cubic
Skip truncation of http(s) URLs in formatted output so links stay usable across styled and markdown, for both object and table views.

- **Bug Fixes**
  - Update `formatCell` to skip truncation for `http://`/`https://` and reject values with spaces via `isURL()` to avoid false positives.
  - Applies to all render paths that funnel through `formatCell`.
  - Add tests: long URL preserved, non-URL truncated, short URL passthrough, whitespace-in-URL rejected, and styled/markdown object/table regressions.

<sup>Written for commit 8d3d5f875511dcbef26ff14be4dd86c046298ec0. Summary will update on new commits.</sup>

### CODE REVIEW

File: internal/output/render.go [hunk 1]
```diff
@@ -670,8 +670,9 @@ func formatCell(val any) string {
 		if strings.ContainsAny(v, "\n\r") {
 			v = strings.Join(strings.Fields(v), " ")
 		}
-		// Truncate long strings (rune-safe for multi-byte UTF-8)
-		if utf8.RuneCountInString(v) > 40 {
+		// Truncate long strings (rune-safe for multi-byte UTF-8).
+		// HTTP(S) URLs are never truncated — a truncated URL is useless.
+		if utf8.RuneCountInString(v) > 40 && !isURL(v) {
```
[the remark this answers is not in the document]
Comment (jeremy) [reply internal/output/render.go:675]: Acknowledged — this is a known trade-off. `selectColumns` still budgets each column at max 40, so column selection itself is unchanged. The only effect is that lipgloss auto-sizes from untruncated content, so styled tables may render wider than the terminal when URL columns are present. A truncated URL is never useful, so correctness wins here. If the wider tables cause real UX pain we'll address table layout separately.

File: internal/output/render.go [hunk 2]
```diff
@@ -733,6 +734,10 @@ func formatCell(val any) string {
 	}
 }
 
+func isURL(s string) bool {
+	return strings.HasPrefix(s, "https://") || strings.HasPrefix(s, "http://")
+}
```
[the remark this answers is not in the document]
Comment (jeremy) [reply internal/output/render.go:740]: Fixed — `isURL` now rejects strings containing spaces, so a newline-collapsed value like `"https://example.com (extra text...)"` is correctly treated as non-URL and still gets truncated. Added test coverage for this edge case.
